scripts/remove_dups.py

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/python3
+#
+# remove_dups.py
+
+import json, os
+
+links = []
+parent = os.path.split(os.path.abspath(os.getcwd()))[0]
+data = open(os.path.join(parent, 'data/data.json'), 'r')
+config = json.load(data)
+data.close()
+
+for paper in config:
+    links.append(paper['Link'])
+
+for link in links:
+    count = 0
+    for paper in config:
+        if link == paper['Link']:
+            count += 1
+
+        if count > 1:
```

Comment:
OUCH! :smile: This should be indented inside. Otherwise, once there is a match it will remove every object. I ran this script and it pruned 4000 objects at once. Careful, @TheMousePotato :slightly_smiling_face:

Reply:
Oh! I'm so sorry about that. Did that in a hurry.